deepdiff/deepdiff.py

```diff
@@ -119,8 +119,8 @@ class DeepDiff(dict):
         >>>
         >>> print (ddiff['values_changed'][0])
         root[4]['b']:
-        --- 
-        +++ 
+        ---
```

Comment:
Please put back the spaces after `---` and `+++`. It sounds stupid but they are needed for docstrings to pass. Basically the difflib is printing `--- ` (with spaces) and not `---`